src/assets/Dashboard.js

```diff
@@ -55,15 +55,15 @@ const Dashboard = (props) => {
               }}
             >
               <Switch>
-                <Route
+                <PrivateRoute
                   path={`${path}/room/:roomID`}
                   component={RoomContent}
-                ></Route>
-                <Route
+                ></PrivateRoute>
+                <PrivateRoute
                   exact
                   path={`${path}/`}
                   component={DashboardContent}
-                ></Route>
+                ></PrivateRoute>
```

Comment:
Can all of the above PrivateRoutes be self closing tags? Would be cleaner.

Reply:
No problem all done